kickstart/provision_atomic.erb

```diff
@@ -18,7 +18,9 @@ bootloader --timeout=3
 text
 
 <% if @host.os.name.match /.*fedora.*/i -%>
-ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=<%= @host.param_true?('atomic-upstream') ? "https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/" : "#{@host.operatingsystem.medium_uri(@host)}/content/repo/" %> --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
+# Use @host.operatingsystem.medium_uri(@host)}/content/repo/ as the URL if you
+# have set up a local installation media for Fedora
+ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=https://dl.fedoraproject.org/pub/fedora/linux/atomic/#{@host.os.major}/ --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
```

Comment:
If you can deploy your own repository for kickstart trees as I explain here http://blog.daniellobato.me/unattended-deployments-of-fedora-and-rhel-atomic-with-foreman/ , then yeah. That used to be the only option to provision Atomic. I think it's fine to default to use the Fedora URL, and if you set up your own Atomic installation media then change the template. 

There's one comment in lines 21 and 22 with the command to use as the URL if there's a local Fedora Atomic installation media.